aggie-coding-club/Rev-Registration
# Update README

## Description

This updates the README w/ a description, features list, and demo gif.

## How to test

You can check out what it would look like by clicking the most recent commit and then clicking "Browse Files", or by going [here](https://github.com/aggie-coding-club/Rev-Registration/tree/11f0152)

## Related tasks

Closes #280
